.github/workflows/build_container.yml

```diff
@@ -0,0 +1,48 @@
+name: Create and publish a container image
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  IMAGE_NAME: fooocus
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository_owner }}/${{ env.IMAGE_NAME }}
```

Comment:
```suggestion
          images: ghcr.io/${{ github.repository_owner }}/${{ github.event.repository.name }}
```

Reply:
GHCR supports only lowercase image names unfortunately, and the repository name is uppercase. I'll merge this for now, as I believe the docker action should make the inputs lowercase automatically, but if not, we're going to have to revert this change.